src/tests/access/test_dual_ownable.cairo

```diff
@@ -20,14 +20,14 @@ use starknet::testing::set_contract_address;
 fn setup_snake() -> (DualCaseOwnable, IOwnableDispatcher) {
     let mut calldata = array![];
     calldata.append_serde(OWNER());
-    let target = utils::deploy(SnakeOwnableMock::TEST_CLASS_HASH, calldata);
+    let target = declare_and_deploy("SnakeOwnableMock", calldata);
```

Comment:
```suggestion
    let target = utils::declare_and_deploy("SnakeOwnableMock", calldata);
```
For consistency, applies for the rest of the module as well.

Reply:
Addresses